.github/workflows/main.yml

```diff
@@ -0,0 +1,226 @@
+---
+name: stratisd CI
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - master
+      - develop-2.2.1
+  pull_request:
+    branches:
+      - master
+      - develop-2.2.1
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # MANDATORY CHECKS USING CURRENT DEVELOPMENT COMPILER
```

Comment:
This matrix combines both checks w/ CURRENT DEVELOPMENT and checks with STABLE. Can you move "CURRENT DEVELOPMENT" comment down to line 24 and place old "STABLE" comment at line 30?